deploy/vagrant/conf/ec2.yml

```diff
@@ -58,6 +58,11 @@ Block_Device_Mapping: []
 # modify only when you know what you are doing
 #
 
+# If you want to use "Spot Instance"
+# (http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-spot-instances.html), 
+# set this field. If this field is not empty, it means you want to use spot instance.
```

Comment:
I think it would be nice to have some clarifications:
- an example of what a "Spot_Price" looks like
- "spot instances will be used if and only if the 'Spot_Price' is not empty."